src/game/AI/ScriptDevAI/scripts/kalimdor/darkshore.cpp

```diff
@@ -827,47 +865,30 @@ struct npc_corrupted_furbolgAI : public ScriptedAI
             return;
 
         m_creature->GetMotionMaster()->MoveIdle();
-        m_isMovementActive  = false;
-        m_purifyTimer = 5 * IN_MILLISECONDS;
+        float angle = m_creature->GetAngle(bowlCoords.GetPositionX(), bowlCoords.GetPositionY());
+        m_creature->SetOrientation(angle);
+        m_creature->SendHeartBeat();
```

Comment:
This might be unnecessary as they walk straight from where they were towards the nearest place on the bounding radius of the bowl, so they don't need to turn around. Sometimes Orientation gets weird in unexpected ways tho, so I'm not sure about it.
SendHeartBeat(); is required for some reason for the creature's orientation to be sent to clients/players, otherwise the orientation can be undefined clientside, even if well defined serverside